apps/files/src/components/NavigationQuota.vue

```diff
@@ -88,8 +88,8 @@ export default {
 	},
 
 	mounted() {
-		// Warn the user if the available storage is 0 on page load
-		if (this.storageStats?.free <= 0) {
+		// Warn the user if the available storage is 0 on page load, except for users with 0 B quota
+		if (this.storageStats?.free <= 0 && this.storageStats?.quota != 0) {
```

Comment:
```suggestion
		// Warn the user if the available storage is 0 on page load, except for users with 0 B quota
		if (this.storageStats?.free <= 0 && this.storageStats?.quota != 0 && this.storageStats?.quota != -3) {
```
I think something like the above can fix #43280 here too.

When the quota is unlimited, `quota` should be `-3` here e.g. [SPACE_UNLIMITED](https://github.com/nextcloud/server/blob/1eef71eaf908f9e071fff3d44f5c162aa3a60e08/lib/public/Files/FileInfo.php#L63) via [`getStorageInfo()`](https://github.com/nextcloud/server/blob/1eef71eaf908f9e071fff3d44f5c162aa3a60e08/lib/private/legacy/OC_Helper.php#L515C5-L515C5) and [`getUserQuota()`](https://github.com/nextcloud/server/blob/1eef71eaf908f9e071fff3d44f5c162aa3a60e08/lib/private/legacy/OC_Util.php#L166-L168) et al.

Reply:
yes, I think this makes sense